src/Chapter_03_Linked_Lists/DoublyLinkedList.c

```diff
@@ -180,7 +180,7 @@ void delete(struct DLLNode **head, int position) {
         temp = temp->next;
         k++;
     }
-    if(k < position-1){
+    if(k < position){
```

Comment:
I think @Mishra-Sagar ,
**`if( k != position) `**  is also valid . What do you say?